src/editor.js

```diff
@@ -1729,6 +1729,38 @@ Editor.$uid = 0;
         }
     };
 
+    /**
+     * Finds link at defined {row} and {column}
+     * @returns {String}
+     **/
+    this.findLinkAt = function (row,column) {
+        var line = this.session.getLine(row);
+        var wordParts = line.split(/((?:https?|ftp):\/\/[\S]+)/);
+        var delta = column;
+        if (delta < 0) delta = 0;
+        var curLength = 0, itLength = 0, match;
+        wordParts.forEach(function (item) {
+            itLength = curLength + item.length;
+            if (delta >= curLength && delta <= itLength) {
+                if (item.match(/((?:https?|ftp):\/\/[\S]+)/)) {
+                    match = item.replace(/[\s:.,'";}\]]+$/, "");
```

Comment:
should we add break here so we don't look at the rest of the line? it can help with large lines